cmd/serve/serve.go

```diff
@@ -34,13 +34,17 @@ var ServeCmd = &cobra.Command{
 			backend = os.Getenv("K8SGPT_BACKEND")
 			password := os.Getenv("K8SGPT_PASSWORD")
 			model := os.Getenv("K8SGPT_MODEL")
+			baseURL := os.Getenv("K8SGPT_BASEURL")
+			engine := os.Getenv("K8SGPT_AZENGINE")
 			// If the envs are set, alocate in place to the aiProvider
 			// else exit with error
-			if backend != "" || password != "" || model != "" {
+			if backend != "" || password != "" || model != "" || baseURL != "" || engine != "" {
```

Comment:
small nit, feel free to ignore as it's just about style: since this is starting to grow, maybe for better visibility makes sense to have a var for it ?

For instance:

```go
envIsSet := backend != "" || password != "" || model != "" || baseURL != "" || engine != ""

if envIsSet {
 // ....
}
```